data-builder/parse_augments.py

```diff
@@ -29,14 +29,19 @@ def parse_augments():
             level = row[0].value
             value = row[3].value
             bonusType = row[4].value
+            description = row[5].value
```

Comment:
I started to incorporate the notion of a "description" to go along with each affix, but I was not yet able to get it fully connected up throughout the application. I think it would be awesome if the site could show tooltips when you hover over terms that describes what they do.